x-pack/plugins/enterprise_search/public/applications/workplace_search/index.test.tsx

```diff
@@ -68,12 +68,6 @@ describe('WorkplaceSearchConfigured', () => {
     expect(mockKibanaValues.renderHeaderActions).toHaveBeenCalledWith(WorkplaceSearchHeaderActions);
   });
 
-  it('initializes app data with passed props', () => {
-    shallow(<WorkplaceSearchConfigured isFederatedAuth />);
-
-    expect(initializeAppData).toHaveBeenCalledWith({ isFederatedAuth: true });
-  });
-
```

Comment:
I would prefer we didn't delete this test entirely as it's still valid / we're still passing props, just not specifically `isFederatedAuth`. If looks like all WS is passing now is workplace search data, so maybe something like this:

```js
import { DEFAULT_INITIAL_APP_DATA } from '../../../common/__mocks__';

  it('initializes app data with passed props', () => {
    const { workplaceSearch } = DEFAULT_INITIAL_APP_DATA;
    shallow(<WorkplaceSearchConfigured workplaceSearch={workplaceSearch} />);

    expect(initializeAppData).toHaveBeenCalledWith({ workplaceSearch });
  });
```

Reply:
Good call. Thanks! https://github.com/elastic/kibana/pull/103278/commits/a46f82620af2a578962f82b2b651f3368eda3a59